scalableminds/webknossos
# Infinite admin tasks

### Mailable description of changes:
 - infinite tasks for admins

### Steps to test:
- create two teams and a user, who is admin in one of the teams
- create ~10 tasks per team
- try to get as many tasks as possible
- you should get infinite (~10) tasks from one and 5 tasks from the other project

### Issues:
- fixes #2042

------
